pytest_xmlcomp/hooks.py

```diff
@@ -0,0 +1,7 @@
+"""Custom hooks for pytest-xmlcomp"""
+
+
+def pytest_transform_XML(xmlfile):
+    """Hook for transforming XML"""
+    print("Given XML file: %s", xmlfile)
```

Comment:
For the time being, you can leave it as is, but when we create a release, it should be removed.